Release capa v2.0 🎉 (#671)
fireeye/capa: CHANGELOG.md

## master (unreleased)

### New Features

### Breaking Changes

### New Rules (0)

-

### Bug Fixes

### capa explorer IDA Pro plugin

### Development

### Raw diffs
- [capa v2.0.0...master](https://github.com/fireeye/capa/compare/v2.0.0...master)
- [capa-rules v2.0.0...master](https://github.com/fireeye/capa-rules/compare/v2.0.0...master)

## v2.0.0 (2021-06-30)

We are excited to announce version 2.0! :tada:

> capa 2.0 enables anyone to contribute rules more easily and it is the first Python 3 ONLY version.
> It includes many features and enhancements for the capa explorer IDA plugin and more concise and relevant result via identification of library functions using FLIRT.
> It also adds 89 new rules, including all new techniques introduced in MITRE ATT&CK v9.

Reviewer: ```suggestion
capa 2.0:
- enables anyone to contribute rules more easily
- is the first Python 3 ONLY version
- includes many features and enhancements for the capa explorer IDA plugin
- provides more concise and relevant result via identification of library functions using FLIRT
- adds 89 new rules, including all new techniques introduced in MITRE ATT&CK v9
```

## master (unreleased)

### New Features

### Breaking Changes

### New Rules (0)

- 

### Bug Fixes

### capa explorer IDA Pro plugin

### Development

### Raw diffs
- [capa v2.0.0...master](https://github.com/fireeye/capa/compare/v2.0.0...master)
- [capa-rules v2.0.0...master](https://github.com/fireeye/capa-rules/compare/v2.0.0...master)

## v2.0.0 (2021-06-30)

We are excited to announce version 2.0! :tada:
capa 2.0:
- enables anyone to contribute rules more easily
- is the first Python 3 ONLY version
- includes many features and enhancements for the capa explorer IDA plugin
- provides more concise and relevant result via identification of library functions using FLIRT
- adds 89 new rules, including all new techniques introduced in MITRE ATT&CK v9

> We appreciate everyone who opened issues, provided feedback, and contributed code and rules.

Reviewer: ```suggestion
A huge thanks to everyone who submitted issues, provided feedback, and contributed code and rules. Many colleagues across dozens of organizations have volunteered their experience to improve this tool!
```
